src/utils/event.ts

```diff
@@ -60,11 +60,21 @@ export const eventHappeningNow = (
 export const hasEventFinished = (event: VenueEvent) =>
   isAfter(Date.now(), eventEndTime(event));
 
-export const eventStartTime = (event: VenueEvent) =>
-  fromUnixTime(event.start_utc_seconds);
+export const eventStartTime = (event: VenueEvent) => {
+  if (!event?.start_utc_seconds) {
```

Comment:
had some events with missing details, so had to add this guard to prevent timing page from breaking

Reply:
Will this impact other calls of the same utility function?

Reply:
Consider this: pure function got turned into impure one.
Now none of  the places it is called from can depend on it to consistently return the same result for the same input.

A way to make it pure again is to not generate input inside, but add second options parameter with field `defaultValue` for the places that want this kind of behavior

Reply:
Or, just fix your events ;)